### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Weyland2093**
Traduction française de l'application ! ;-)

### CODE REVIEW

File: src/renderer/i18n/fr/wallet.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  'wallet.imports.keystore.select': 'sélectionner le fichier clé privée',
+  'wallet.imports.keystore.upload': 'Choisir le fichier à uploader',
+  'wallet.imports.phrase': 'Phrase',
+  'wallet.imports.wallet': 'Importer un wallet existant',
+  'wallet.imports.enterphrase': 'Entrez la phrase',
+  'wallet.imports.error.instance': 'Impossible de créer une instance du Client',
+  'wallet.imports.error.keystore.load': 'Clé privée incorrecte',
+  'wallet.imports.error.keystore.import': 'Mot de passe incorrect',
+  'wallet.phrase.error.valueRequired': 'La phrase est requise',
+  'wallet.phrase.error.invalid': 'Phrase incorrecte',
```
Comment (mcadam) [inline src/renderer/i18n/fr/wallet.ts:33]: Phrase de récupération incorrecte

File: src/renderer/i18n/fr/wallet.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  'wallet.phrase.error.import': "Erreur lors de l'importation de la phrase",
```
Comment (mcadam) [inline src/renderer/i18n/fr/wallet.ts:34]: Erreur lors de l'importation de la phrase de récupération

**Review by mcadam — CHANGES_REQUESTED** [review]

File: src/renderer/i18n/fr/common.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  'common.from': 'De',
+  'common.amount': 'Montant',
+  'common.coin': 'Coin',
+  'common.password': 'Mot de passe',
+  'common.memo': 'Mémo',
+  'common.date': 'Date',
+  'common.refresh': 'Rafraîchir',
+  'common.back': 'Retour',
+  'common.remove': 'Supprimer',
+  'common.keystore': 'Clé privée',
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/common.ts:21]: https://ibb.co/gyb3FC8

File: src/renderer/i18n/fr/common.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  'common.confirm': 'Confirmer',
+  'common.cancel': 'Annuler',
+  'common.next': 'Suivant',
+  'common.finish': 'Finir',
+  'common.copy': 'Copier',
+  'common.loading': 'Chargement...',
+  'common.error': 'Erreur',
+  'common.success': 'Succès',
+  'common.change': 'Modifier',
+  'common.wallet': 'Wallet',
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/common.ts:33]: https://ibb.co/gyb3FC8

[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/common.ts:33]: The french community vote massively for wallet and not Portefeuille...

File: src/renderer/i18n/fr/bonds.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { BondsMessages } from '../types'
+
+const bonds: BondsMessages = {
+  'bonds.node': 'Nœud',
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/bonds.ts:4]: OK for Node.

File: src/renderer/i18n/fr/common.ts [hunk 6]
```diff
@@ -0,0 +1,63 @@
+import { CommonMessages } from '../types'
+
+const common: CommonMessages = {
+  'common.stats': 'Statistiques',
+  'common.network': 'Réseau',
+  'common.faqs': 'FAQ',
+  'common.greeting': 'Hello {name}',
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/common.ts:7]: hummm ok

File: src/renderer/i18n/fr/common.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/common.ts:21]: We can keep keystore : https://fr.wikipedia.org/wiki/Keystore

File: src/renderer/i18n/fr/common.ts [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  'common.max': 'Maximum',
+  'common.search': 'Rechercher',
+  'common.retry': 'Réessayer',
+  'common.reload': 'Recharger',
+  'common.add': 'Ajouter',
+  'common.swap': 'Swap',
+  'common.liquidity': 'Liquidité',
+  'common.step': 'Étape {current}/{total}',
+  'common.done': 'Terminé',
+  'common.thorAddress': 'THORAdresse',
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/common.ts:52]: ok

File: src/renderer/i18n/fr/deposit.ts [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+  'deposit.add.asym': 'Ajouter {asset}',
+  'deposit.add.sym': 'Ajouter {assetA} + {assetB}',
+  'deposit.add.state.sending': 'Envoi de la transaction de dépôt',
+  'deposit.add.state.checkResults': 'Vérification du résultat du dépôt',
+  'deposit.add.state.pending': 'Ajout de liquidité',
+  'deposit.add.state.success': 'Dépôt réussi',
+  'deposit.add.state.error': 'Erreur lors du dépôt',
+  'deposit.add.error.chainFeeNotCovered': 'Les frais nécessaires de {fee} ne sont pas couverts par votre solde : {balance}',
+  'deposit.add.error.nobalances': 'Aucun solde',
+  'deposit.add.error.nobalance1': "Vous n'avez pas de {asset} à déposer dans votre wallet.",
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/deposit.ts:26]: Not agree, in the french community we all use the term : wallet

File: src/renderer/i18n/fr/deposit.ts [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+  'deposit.add.error.nobalance2': "Vous n'avez pas de {asset1} et de {asset2} à déposer dans votre wallet.",
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/deposit.ts:27]: no

File: src/renderer/i18n/fr/deposit.ts [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+  'deposit.bond.state.error': 'Erreur de caution',
+  'deposit.unbond.state.error': 'Erreur de retrait',
+  'deposit.leave.state.error': 'Erreur de sortie',
+  'deposit.withdraw': 'Récupérer',
+  'deposit.advancedMode': 'Mode avancé',
+  'deposit.drag': 'Glisser pour déposer',
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/deposit.ts:33]: sounds a bit lame

File: src/renderer/i18n/fr/deposit.ts [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+  'deposit.pool.noDeposit': "Vous ne possédez aucune part dans cette pool",
+  'deposit.withdraw.pending': 'Retirer',
+  'deposit.withdraw.success': 'Retrait réussi',
+  'deposit.withdraw.error': 'Erreur de retrait',
+  'deposit.withdraw.title': 'Ajuster le retrait',
+  'deposit.withdraw.choseText': 'Choisissez de 0 à 100% la part du retrait.',
+  'deposit.withdraw.receiveText': 'Vous devriez recevoir.',
+  'deposit.withdraw.fees': 'Frais de transaction : {thorMemo}, Frais de retrait : {thorOut} + {assetOut}',
+  'deposit.withdraw.feeNote': 'Remarque: {fee} BNB seront laissés dans votre wallet pour les frais de transaction.',
+  'deposit.withdraw.drag': 'Glisser pour retirer',
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/deposit.ts:50]: Pas d'accord. "Glisser pour retirer" fonctionne.

File: src/renderer/i18n/fr/midgard.ts [hunk 12]
```diff
@@ -0,0 +1,8 @@
+import { MidgardMessages } from '../types'
+
+const midgard: MidgardMessages = {
+  'midgard.error.byzantine.title': 'Erreur Byzantine Midgard',
+  'midgard.error.byzantine.description': "Impossible d'atteindre le consensus en tentant de trouver un points d'accès à l'API Midgar"
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/midgard.ts:5]: ok

File: src/renderer/i18n/fr/pools.ts [hunk 13]
```diff
@@ -0,0 +1,14 @@
+import { PoolsMessages } from '../types'
+
+const pools: PoolsMessages = {
+  'pools.depth': 'Profondeur',
+  'pools.24hvol': 'volume sur 24h',
```
[the remark this answers is not in the document]
Comment (Weyland2093) [reply src/renderer/i18n/fr/pools.ts:5]: No majuscule in the english file. But ok i change.

File: src/renderer/i18n/fr/common.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (mcadam) [reply src/renderer/i18n/fr/common.ts:33]: OK yeah I took a look at what others were doing like Binance on Binance.com or in trust wallet and other apps and they all translated it to Portefeuille

File: src/renderer/i18n/fr/deposit.ts (hunk 10 above)
[the remark this answers is not in the document]
Comment (mcadam) [reply src/renderer/i18n/fr/deposit.ts:33]: Checked what they used in iPhone Lock Screen at the time when they had the slider but don't feel strongly about it lol

File: src/renderer/i18n/fr/deposit.ts (hunk 11 above)
[the remark this answers is not in the document]
Comment (mcadam) [reply src/renderer/i18n/fr/deposit.ts:50]: Why not

**Review by mcadam — APPROVED** [review]

**Comment by veado** [comment]
@Weyland2093 Thanks for your contribution!

**Pull request merged**